README.md

```diff
@@ -82,6 +82,27 @@ const App = () => (
 );
 ```
 
+## Disable data-rh attribute
+
+`<HeadProvider/>` accepts an optional prop `whitelist` which accepts a comma separated CSS selector string just like `document.querySelector`. This allows `react-head` to know which tags it controlls without the need for the `data-rh` attribute, which might lead to problems with some HTML parsers of SEO tools for example.
+
+### Example
+
+```jsx
+// on the server
+<HeadProvider headTags={headTags} whitelist="title,[name="description"],[property^="og:"]">
```

Comment:
Would it make sense to have `whitelist` API accept an array instead of comma-separated string?

Reply:
Glad you like it 😃 Yea I thought about this as well, what do you think, array only or both?

Reply:
I think Array only to simplify the implementation code on our side